api/v1/collector_server.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package llmariner.apiusage.server.v1;
+
+option go_package = "github.com/llmariner/api-usage/api/v1";
+
+message UsageRecord {
+  string user = 1;
+  string tenant = 2;
+  string organization = 3;
+  string project = 4;
+
+  int64 timestamp = 5;
+  int32 latency_ms = 6;
```

Comment:
not sure about the exact usage, but we might need to revisit this and define time_to_first_token and time_per_output_token for inference (https://www.databricks.com/blog/llm-inference-performance-engineering-best-practices).